.npmrc

```diff
@@ -0,0 +1 @@
+auto-install-peers=true
```

Comment:
Can we remove this please? Does it work as expected if we do?

Reply:
Removed. This actually wasn't doing anything, turns out. Thanks for pointing out.